cloudsql/mysql/database-sql/cloudsql_test.go

```diff
@@ -22,20 +22,55 @@ import (
 	"testing"
 )
 
+type commonTestInfo struct {
+	dbName                 string
+	dbPass                 string
+	dbUser                 string
+	dbPort                 string
+	instanceConnectionName string
+}
+
+func getCommonTestInfo() commonTestInfo {
+	return commonTestInfo{
+		dbName:                 os.Getenv("MYSQL_DATABASE"),
+		dbPass:                 os.Getenv("MYSQL_PASSWORD"),
+		dbPort:                 os.Getenv("MYSQL_PORT"),
+		dbUser:                 os.Getenv("MYSQL_USER"),
+		instanceConnectionName: os.Getenv("MYSQL_INSTANCE"),
+	}
+}
+
 func TestIndex(t *testing.T) {
 	if os.Getenv("GOLANG_SAMPLES_E2E_TEST") == "" {
 		t.Skip()
 	}
+
+	_commonTestInfo := getCommonTestInfo()
+
 	tests := []struct {
 		dbHost string
+		common commonTestInfo
```

Comment:
Does this need to be included? Can we just use the var directly?

Reply:
In my limited knowledge of Go, this pattern was removing the need to declare all of the shared variables twice. Is there an even simpler way?

Reply:
Ohh wait I see what you mean now. Updating!